### ISSUE TITLE
Question - test:after:run event

### ISSUE THREAD

**Issue by GabRog** [javierbrea/cypress-fail-fast#61]
I am adding context to display screenshots and videos from failed test.
```
Cypress.on('test:after:run', (test, runnable) => {
  if (test.state === 'failed') {
    addContext({ test }, `screenshots/${Cypress.spec.name}/${runnable.parent.title} -- ${test.title} (failed).png``);
    addContext({ test },  `videos/${Cypress.spec.name}.mp4`);
  }
});
```
However when cypress-fast-fail plugin is enabled, adding context is not executed at all. 
Could you tell me what am I doing wrong or provide some workaround for this issue?

**Comment by javierbrea** [comment]
Hi @GabRog,

I will investigate it to fix it or provide a workaround as soon as possible.

Thanks for your feedback! 🙂

**Comment by jmtimko5** [comment]
I saw this issue as well, thanks for investigating @javierbrea

**Comment by javierbrea** [comment]
Hi @GabRog and @jmtimko5,
A solution has been added and published in [release v2.1.0](https://github.com/javierbrea/cypress-fail-fast/releases/tag/v2.1.0). Thanks again for your feedback!

**Comment by javierbrea** [comment]
I'm sorry, but the provided solution seems to not be working fine. E2E tests were added and seemed to be working properly, but I have just realized that are unstable. I will revert the change to make the plugin stable again, and reopen this issue until a better solution is found.

**Comment by javierbrea** [comment]
Finally resolved in [release v2.2.0](https://github.com/javierbrea/cypress-fail-fast/releases/tag/v2.2.0).

**Comment by GabRog** [comment]
Hi, I have a problem with screenshots.

**Comment by GabRog** [comment]
I have still problem with screenshots. 
When I comment line where screenshot are added:
```Cypress.on('test:after:run', (test, runnable) => {
  if (test.state === 'failed') {
    // addContext({ test }, `screenshots/${Cypress.spec.name}/${runnable.parent.title} -- ${test.title} (failed).png``);
    addContext({ test },  `videos/${Cypress.spec.name}.mp4`);
  }
});```
I have in my report video attached. 

When this line is uncommented, video nor screenshots are added to report. 
I tried also with screenshots only - they don't appear in report. 
Can you check if this can be caused by your latest changes?

**Comment by javierbrea** [comment]
Hi @GabRog , I don't think that the plugin may be related now. I added screenshots also while developing the E2E tests and it worked properly (finally I replaced it by adding a simple text just for performance improvement, but the screenshots were working too).

It seems to be a problem just in the line where you are adding the screenshots as context. Have you noticed that you have a double ` character closing the string, for example?
